nimbus-core/src/main/java/com/antheminc/oss/nimbus/domain/model/state/repo/db/MongoSearchByQueryOperation.java

```diff
@@ -56,168 +63,193 @@
  *
  */
 @EnableAPIMetricCollection
-@SuppressWarnings({ "rawtypes", "unchecked"})
+@SuppressWarnings({ "rawtypes", "unchecked" })
```

Comment:
General comment on the entire class. Due to change in indentation a lot of lines of code are showing up as changed. This can cause merge issue. Can you please retain the old indentation